lib/r10k/source/git.rb

```diff
@@ -111,9 +117,25 @@ def desired_contents
 
   private
 
+  def filter_branches(branches, ignore_prefixes)
+    filter = Regexp.new("^(#{ignore_prefixes.join('|')}).?")
+    branches = branches.select do |branch|
```

Comment:
i think if you use `reject` (https://ruby-doc.org/core-2.4.2/Enumerable.html#method-i-reject) instead of `select` here you can avoid the negation of `result` below.

Reply:
Changing.